setup.py

```diff
@@ -20,29 +20,53 @@
 # 		   http://ilastik.org/license/
 ###############################################################################
 from setuptools import setup
+import setuptools_scm
 
 packages = [
     "volumina",
+    "volumina._testing",
+    "volumina.icons",
     "volumina.pixelpipeline",
+    "volumina.pixelpipeline.datasources",
+    "volumina.pixelpipeline.imagesources",
     "volumina.skeletons",
-    "volumina.widgets",
+    "volumina.tiling",
     "volumina.utility",
     "volumina.view3d",
-    "volumina._testing",
+    "volumina.widgets",
 ]
 
 package_data = {
-    "volumina.widgets": ["*.ui", "ui/*.ui"],
     "volumina": ["*.ui"],
-    "volumina._testing": ["*.tif", "*.png", "lena.npy", "rgba129x104.npy"],
+    "volumina._testing": ["*.npy", "*.txt"],
+    "volumina.icons": ["*"],
+    "volumina.view3d": ["ui/*.ui"],
+    "volumina.widgets": ["*.ui", "ui/*.ui"],
 }
 
+_version = setuptools_scm.get_version(write_to="volumina/_version.py")
```

Comment:
When configuring a "normal" Python package with `setuptools_scm`, it's possible to omit `version` from `setup.py`/`setup.cfg` entirely, and do `__version__ = importlib.metadata.version("volumina")` in `volumina/__init__.py` (for Python < 3.8 there is a [backport](https://pypi.org/project/importlib-metadata/)).

Would this work with the conda setup?

Reply:
I didn't get this working. Really it was the only way to duplicate it.